action.yaml

```diff
@@ -59,6 +59,14 @@ inputs:
     description: Ignore unfixed vulnerabilities (default "false")
     required: false
     default: "false"
+  trivy-db-repository:
+    description: OCI repository(ies) to retrieve trivy-db in order of priority
+    required: false
+    default: "ghcr.io/aquasecurity/trivy-db:2,public.ecr.aws/aquasecurity/trivy-db:2"
+  trivy-java-db-repository:
+    description: OCI repository(ies) to retrieve trivy-java-db in order of priority
+    required: false
+    default: "ghcr.io/aquasecurity/trivy-java-db:1,public.ecr.aws/aquasecurity/trivy-db:1"
```

Comment:
Should these options be customizable by users?
If so, I think we should add these to `All options` section in README as well.

Reply:
I updated readme.
9c9c949d45555428770ec75561f10ee21c08be35